Makefile

```diff
@@ -13,7 +13,8 @@ install:
 	@make .installed
 
 .installed: Pipfile Pipfile.lock
-	@echo "Pipfile(.lock) is newer than .installed, (re)installing"
+	@echo "Pipfile(.lock) is newer than .installed, (re)installing in 3 seconds..."
+	@sleep 3
```

Comment:
Can we do without this? Or decrease to 1?

Reply:
Removed it.